docker/build/Dockerfile

```diff
@@ -13,6 +13,13 @@ RUN curl -fsSL https://download.docker.com/linux/debian/gpg | gpg --dearmor -o /
 RUN apt update && apt install -y docker-ce docker-ce-cli containerd.io \
     && apt-get clean
 
+RUN apt-get install apt-transport-https ca-certificates
```

Comment:
Any reason for not adding this in the first `apt-get install` above (line 7)?

Also, remove the `apt-get clean` above (line 14) and move it to after the last `apt-get install` below.

Reply:
Hi @julio-lopez ,
Thank you for the review, I can work on above suggestions.
But I initially thought that we would just need to install gcloud and gcloud auth plugin, in the kanister build image, and things would work but that doesn't seem to be the case.

We will also have to set up gcloud authentication, which I think needs to be discussed with the team and then worked on, so I am planning to park this for now and work on it after discussing this in community call.